UnityProject/Assets/Scripts/UI/Systems/Lobby/AccountLoginPanel.cs

```diff
@@ -26,6 +26,9 @@ public class AccountLoginPanel : MonoBehaviour
 		[SerializeField]
 		private Button exitButtonControl = default;
 
+		[SerializeField]
+		private Button OffLineButtonControl = default;
```

Comment:
could this be a property instead so we need no setter method like we were typing java?

Reply:
it set in the unity editor so I don't think editor likes properties

Reply:
wrong field, sorry